Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -424,10 +423,11 @@ public extension Persistence {
                 }
 
                 filesToRemove.forEach {
-                    guard let _ = try? self.remove(fileAtURL: $0.url, size: $0.fileAttr.size) else {
-                        assertionFailure("💥 Failed to remove file with path 👉 \($0)")
-
-                        return print("💥[Alicerce.Persistence.DiskMemoryPersistenceStack]: Failed to remove file with path 👉 \($0)")
+                    do {
+                        try self.remove(fileAtURL: $0.url, size: $0.fileAttr.size)
```

Comment:
Is this `self.` necessary?

Reply:
Yes, because we're inside a `DiskMemoryBlockOperation` execution block 😉